hub-templates/daskhub/values.yaml

```diff
@@ -116,6 +92,29 @@ basehub:
                   break
           else:
               print("dask-gateway service not found. Did you set jupyterhub.hub.services.dask-gateway.apiToken?")
+        daskhub-02-cloud-storage-bucket: |
+          from z2jh import get_config
+          cloud_resources = get_config('cloudResources')
+          scratch_bucket = cloud_resources['scratchBucket']
+          import os
+
+          if scratch_bucket['enabled']:
+            # FIXME: Support other providers too
+            assert cloud_resources['provider'] == 'gcp'
+            project_id = cloud_resources['gcp']['projectId']
+
+            release = os.environ['HELM_RELEASE_NAME']
+            bucket_protocol = 'gcs'
+            bucket_name = f'{project_id}-{release}-scratch-bucket'
+            env = {
+              'SCRATCH_BUCKET_PROTOCOL': bucket_protocol,
+              # Matches "daskhub.scratchBUcket.name" helm template
+              'SCRATCH_BUCKET_NAME': bucket_name,
+              'SCRATCH_BUCKET': f'{bucket_protocol}://{bucket_name}',
```

Comment:
From the deleted lines, it seems the `JUPYTERHUB_USER` is needed at the end:

`          value: $(SCRATCH_BUCKET_PROTOCOL)://$(SCRATCH_BUCKET_NAME)/$(JUPYTERHUB_USER)`

Maybe I am missing something here and somehow that is already included in the `bucket_name`?

Reply:
Good catch, @damianavila! Fixed :)